src/js/main.es6

```diff
@@ -0,0 +1,77 @@
+$(document).ready(function () {
+  let modal = function Modal() {
```

Comment:
it is constructor function but as a singleton, I want just one instance of it :D